### DISCUSSION TITLE
📅 Month of Learning 📚  - Copilot 🤖 Community Check-in

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#59644] [Copilot News and Announcements]
We surveyed 500 U.S.-based developers about how managers should consider developer productivity, collaboration, and AI coding tools. This survey reveals [AI’s impact on the developer experience](https://github.blog/2023-06-13-survey-reveals-ais-impact-on-the-developer-experience/) powered by GitHub Copilot. 🚀

Not just that, this week at the Collision Conference we unveiled breaking new research on the [economic and productivity impact of generative AI–powered developer tools](https://github.blog/2023-06-27-the-economic-impact-of-the-ai-powered-developer-lifecycle-and-lessons-from-github-copilot/). The research found that the increase in developer productivity due to AI could boost global GDP by over $1.5 trillion.

--- 

👋 Hello again and welcome to another Copilot community check-in. As you might've seen in the title - this check-in is packed with resources that'll help you learn Copilot effectively. Let’s get started by ✨sparkling some of our amazing community members who stood out this past month.

## ✨ Meet the Community Champions: Shoutout 📢  to these members

- A BIG round of applause 👏 for [@danielalyoshin](https://github.com/danielalyoshin) for sharing the [requirements of using Copilot Chat](https://github.com/orgs/community/discussions/57025) and helping so many other users. 
-  💖 Thank You! [@werniq](https://github.com/werniq) for helping [@rpayanm](https://github.com/rpayanm) make them use [Copilot with PhpStorm](https://github.com/orgs/community/discussions/58055). We saw multiple users reporting the error `WARN - #c.i.o.w.i.s.w.StatusBarWidgetsManager` last week but it was just a temporary issue that got resolved after a few hours.
- Short & Sweet! A BIG shoutout 🔈  for [@conrizzo](https://github.com/conrizzo) - They [shared a workaround](https://github.com/orgs/community/discussions/57943) for a Copilot Chat issue some users experienced temporarily.

## How to Setup Copilot Chat in VS Code? ❓ 
 
🖐️ Our utmost priority is ensuring seamless access to the information you seek. Copilot Chat has garnered immense appreciation from developers, who have found it invaluable. To facilitate a smooth initial setup, we have compiled the most frequently asked questions to guide you through the process efficiently.

[@WirelessLife](https://github.com/WirelessLife) (Microsoft Principal Cloud Advocate) posted a step-by-step guide to setup [Copilot Chat in VS Code](https://github.com/orgs/community/discussions/58756). If you have any questions or feedback to share, feel free to post them in the discussion or in the comments below. 

👀  We’ve also added other key Copilot X FAQ’s [to this discussion](https://github.com/orgs/community/discussions/47318#:~:text=%E2%9A%A1,%F0%9F%91%80). 

## ⚡ Readers Delight - Learn from the Experts! 🔖 

- Wanna know how to use GitHub Copilot at its best? ❓  Our Copilot experts have put together a prompt guide for you that explains **[Top Copilot Prompts, tips, and use cases](https://github.blog/2023-06-20-how-to-write-better-prompts-for-github-copilot/)** - This guide will help you write better prompts and get the most out of Copilot. 
- Ever wondered 🤔  how code is generated in AI tools? Explore **"[What is AI code generation](https://resources.github.com/artificial-intelligence/what-is-ai-code-generation/)"** for a satisfying dive into the intricacies of this fascinating technology. Thank us later! 🔙 
- Generative AI, Copilot, blah blah blah 🗨️  - who needs it? You're the Ultimate Programmer, so why would you want some pretentious "large" language model helping you? Read **[Top Reasons NOT to use GitHub Copilot](https://github.com/orgs/community/discussions/58562)** 👀 
  - If you already have that figured out ✅ , here how you should be [Using Copilot Effectively](https://github.com/orgs/community/discussions/54286). 
- [The ReadME Project](https://github.com/readme): We have three guides in our May edition 📖 , contributed by developers like you, on the different ways they use generative AI such as GitHub Copilot for software development. **Wanna get a TL;DR?**  We’ve got you covered in **[Using GitHub Copilot for MLOps, TDD, & more](https://github.com/orgs/community/discussions/57344)** 💯 
- **Is your company using generative AI yet?** [Get to the Full Story!](https://github.blog/2023-05-09-how-companies-are-boosting-productivity-with-generative-ai/) Explore how generative AI coding tools are changing the way developers and companies build software. 🔍 
  - Want to explore the world of generative AI for developers? 🌎 Check out our generative AI guide to learn what it is, how it works, and what it means for developers everywhere.  **[Get the guide >](https://github.blog/2023-04-07-what-developers-need-to-know-about-generative-ai/)**

### 🎉 Bonus! - Prefer Learning on Video? 🎥 

If you have just started your AI learning journey - we have a series on [Learning AI with Github Copilot](https://www.youtube.com/playlist?list=PLlrxD0HtieHiATMDpZkxxQL0YTTiMbcR9)

### 🎊  Add on Bonus! - Training Modules ⌨️ 

From Novice to Pro - **Learn GitHub Copilot from [Microsoft Learn](https://learn.microsoft.com/en-us/training/modules/introduction-to-github-copilot/)** 

- In the [Microsoft Learn module](https://learn.microsoft.com/en-us/training/modules/introduction-to-github-copilot/), you'll learn how to set up [GitHub Copilot](https://github.com/features/copilot), understand how it works, and get started with using it in your own projects. 

Are the resources above helpful to learning GitHub Copilot ? Do you have other resources that serve you your dose 💊 of AI updates? Let us know in the comments below. 🚀 

#### Previously: [⛽ Fueling Collaboration & Knowledge Exchange - Copilot 🤖  Community Check-in](https://github.com/orgs/community/discussions/57420)

**Comment by moonfury-ops** [comment]
Thank you
